docs/migrate-module.md

```diff
@@ -0,0 +1,94 @@
+
+## Before you begin
+
+- If you're running Microsoft Windows, set up Windows Subsystem for Linux [WSL](https://ubuntu.com/wsl) and run commands within WSL.
+- Make sure that you have a current version of Python 3 and pip.
+- Make sure that you have a recent stable version of [Go](https://go.dev/doc/install) installed and available in your PATH environment variable.
+
+## Create an empty module
+
+Create an empty module repo in the `terraform-ibm-modules` GitHub organization.
+
+1.  In the upper right of any page in the [terraform-ibm-modules](https://github.com/terraform-ibm-modules) organization, use the **+** menu, and select **New repository**.
+1.  Select `terraform-ibm-modules` as the owner.
+1.  Enter a name for the module in format `terraform-ibm-<NAME>`, where `<NAME>` reflects the type of infrastructure that the module manages. Use hyphens as delimiters for names with multiple words (for example, terraform-ibm-`activity-tracker`).
+1.  Provide a short description of the module. The description is displayed under the repository title on the [organization page](https://github.com/terraform-ibm-modules) and in the **About** section of the repository. Use the description to help users understand what your repo does by looking at the description.
+1.  Confirm that you are creating a public repository in the terraform-ibm-modules organization and click **Create repository**.
+
+## Clone and initialize the repo
+
+Clone the repo that you just created.
+
+[inc-clone.md](inc-clone.md ':include')
+
+## Migrate the module code
+
+1.  From the root of the newly cloned repo, create a branch to work in by running the command `git checkout -b ＜YOUR_WORKING_BRANCH>`. Replace `＜YOUR_WORKING_BRANCH>` with your own branch name.
+s
+1.  Push the module code from the older repo to the new one by running these commands:
+    1.  Add the new repo as a remote to the older repo:
+
+        ```bash
+        git remote add origin2 https://github.com/terraform-ibm-modules/<REPO_NAME>
+        ```
+
+        Where `<REPO_NAME>` is your new repo. The command uses `origin2` because Git uses `origin` by default.
+    1.  Check that the remote points from your old repo to the new one by running this command:
+
+        ```bash
+        git remote -v
+        ```
+    1.  Push the module code from the older repo to the main branch of the new repo:
+        ```bash
+        git push origin2 main
+        ```
+
+## Add the newer CI/CD code
+
+Copy the following files from the `.github` directory of the terraform-ibm-module-template.
+
+1.  Browse to the [terraform-ibm-module-template](https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/main/.github) `.github` directory.
+1.  Copy the `settings.yml` file.
+1.  Copy the `workflows` directory that contains the GitHub Action workflow
+
+## Make updates to pass the pre-commit checks
```

Comment:
We're missing adding a git submodule to https://github.com/terraform-ibm-modules/common-dev-assets and creating symbolic links (Makefile, ci, Brewfile, .pre-commit-config.yaml – see soft links at https://github.com/terraform-ibm-modules/terraform-ibm-module-template for reference).

This step can be handled by running the script that is coming at https://github.com/terraform-ibm-modules/common-dev-assets/pull/85

Reply:
We have the submodule info. Those steps are pulled in by include (`[inc-clone.md](inc-clone.md ':include')` under `## Clone and initialize the repo`. The PDF attachment in the description might help show the whole topic.

Reply:
This is using an include for both cloning and initializing the submodule. The script doesn't clone, so let me know if it we want to change this to point to the script.

Reply:
The command is different on an empty repo - it would be ` git submodule add https://github.com/terraform-ibm-modules/common-dev-assets` instead of update

Reply:
And then symbolic links need to be created as well

Reply:
Added the 4 commands from the script.